proposed/2022/dotnet-nuget-config-spec.md

```diff
@@ -0,0 +1,358 @@
+# NuGet configuration CLI for dotnet.exe
+
+- Author: [Heng Liu](https://github.com/heng-liu)
+- GitHub Issue [8420](https://github.com/NuGet/Home/issues/8420)
+
+## Problem Background
+
+Currently, there is no NuGet configuration CLI for dotnet.exe. It's inconvenient for NuGet users to know about the NuGet configuration file locations and figure out where is the merged configuration coming from. 
+
+## Who are the customers
+
+This feature is for dotnet.exe users.
+
+## Goals
+Design and implement `dotnet nuget config` command.
+
+## Non-Goals
+Design and implement in the same way for NuGet.exe command.
+
+## Solution
+The following command will be implemented in the `dotnet.exe` CLI.
+
+### `dotnet nuget config`
+
+#### Commands
+
+- Path
+
+List all the paths of NuGet configuration files that will be applied, when invoking NuGet command from the current working directory path.
```

Comment:
Shouldn't the get command be listed here?

Reply:
Thanks! Changed the structures accordingly.